scripts/runTests.ps1

```diff
@@ -12,79 +12,85 @@ Function global:registerSingleTests()
-    registerTest -testname "server_http"
-    registerTest -testname "shell_client" -index "http"
-    registerTest -testname "shell_client" -vst -index "vst"
-    registerTest -testname "shell_client" -http2 -index "http2"
-    registerTest -testname "shell_client_aql" -index "http"
-    registerTest -testname "shell_client_aql" -vst -index "vst"
-    registerTest -testname "shell_client_aql" -http2 -index "http2"
-    registerTest -testname "shell_fuzzer"
-    If ($ENTERPRISEEDITION -eq "On") { registerTest -testname "shell_client_aql" -encrypt -index "encrypt" }
-    registerTest -testname "server_secrets"
-    registerTest -testname "server_permissions"
-    registerTest -testname "server_parameters"
-    registerTest -testname "BackupAuthNoSysTests"
-    registerTest -testname "BackupAuthSysTests"
-    registerTest -testname "BackupNoAuthNoSysTests"
-    registerTest -testname "BackupNoAuthSysTests"
-    registerTest -testname "agency" -weight 2
-    registerTest -testname "agency-restart"
-    registerTest -testname "active_failover"
-    registerTest -testname "arangobench"
-    registerTest -testname "arangosh"
-    registerTest -testname "authentication"
-    registerTest -testname "catch"
-    registerTest -testname "dump"
-    registerTest -testname "dump_authentication"
-    registerTest -testname "dump_jwt"
-    registerTest -testname "dump_maskings"
-    registerTest -testname "dump_multiple"
-    registerTest -testname "dump_no_envelope"
-    registerTest -testname "dump_encrypted"
-    registerTest -testname "dump_with_crashes"
-    registerTest -testname "endpoints"
-    registerTest -testname "export"
-    registerTest -testname "fuerte"
-    registerTest -testname "http_replication" -weight 2
-    registerTest -testname "http_server" -sniff true
-    registerTest -testname "importing"
-    registerTest -testname "ssl_server"
-    registerTest -testname "version"
-    registerTest -testname "audit_client"
-    registerTest -testname "audit_server"
-    registerTest -testname "permissions"
-    registerTest -testname "paths_server"
-#    registerTest -testname "replication2_client"
-#    registerTest -testname "replication2_server"
-    # Note that we intentionally do not register the hot_backup test here,
-    # since it is currently not supported on Windows. The reason is that
-    # the testing framework does not support automatic restarts of instances
-    # and hot_backup currently needs a server restart on a restore operation
-    # on Windows. On Linux and Mac we use an exec operation for this to
-    # restart without changing the PID, which is not possible on Windows.
-    # registerTest -testname "hot_backup"
+    if (Test-Path "$INNERWORKDIR/ArangoDB/tests/test-definitions.txt" -PathType Leaf) {
+        Write-Host "Using test definitions from repo..."
+        C:\Python310\python.exe $INNERWORKDIR/ArangoDB/scripts/generateJenkinsScripts.py $INNERWORKDIR/ArangoDB/tests/test-definitions.txt -f ps1 | Invoke-Expression
```

Comment:
using absolute paths here is a receipy to disaster. other parts of powershell oskar lean on python being in the path; this should be done here as well.

Reply:
Neither `python` nor `python3` worked. I tried it, as you can see in commit [7ee8c13](https://github.com/arangodb/oskar/pull/386/commits/7ee8c13198c13cf0692db248fd87b009445cc630) and [88d10e4](https://github.com/arangodb/oskar/pull/386/commits/88d10e44e131e0a6013295997bf85dd23ce861f6).